parquet/src/encryption/mod.rs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//! Encryption implementation specific to Parquet, as described
+//! in the [spec](https://github.com/apache/parquet-format/blob/master/Encryption.md).
+
+pub(crate) mod ciphers;
+pub(crate) mod decrypt;
```

Comment:
I think we might have gone too far with making things private, it looks like users won't be able to create decryption properties if everything is `pub(crate)`?

Reply:
`FileDecryptionProperties` and `DecryptionPropertiesBuilder` are `pub` but I suppose that is not public enough.
```suggestion
pub mod decrypt;
```